src/lando/api/legacy/api/stacks.py

```diff
@@ -54,12 +55,13 @@
 
 
 @require_phabricator_api_key(optional=True)
-def get(phab: PhabricatorClient, revision_id: str):
+def get(phab: PhabricatorClient, request: HttpRequest, revision_id: str):
     """Get the stack a revision is part of.
 
     Args:
         revision_id: (string) ID of the revision in 'D{number}' format
```

Comment:
This should probably be updated to reflect what the URL dispatcher is actually passing here (i.e., just the number portion of the revision ID.)

Reply:
On second look, the conversion is no longer necessary, as the only time it was used was when converting it back to an int. I will consolidate the functionality into a single revision ID variable.